dali/pipeline/operators/crop/bbox_crop.h

```diff
@@ -50,21 +50,35 @@ class RandomBBoxCrop : public Operator<Backend> {
   using Crop = BoundingBox;
   using BoundingBoxes = std::vector<BoundingBox>;
 
+  struct ProspectiveCrop {
+    bool success;
+    Crop crop;
+    BoundingBoxes boxes;
+    std::vector<int> labels;
+
+    ProspectiveCrop(
+      bool success, const Crop &crop, const BoundingBoxes &boxes, const std::vector<int> &labels) :
+      success(success), crop(crop), boxes(boxes), labels(labels) {}
+
+    ProspectiveCrop() : success(false), crop(Crop::FromLtrb(0, 0, 1, 1)) { }
```

Comment:
As a side-note: It's better to use field initializers - that way the constructors will only replace default initialization if needed and other fields will be consistently initialized. Not that it matters in this particular class.

Reply:
Done